tests/ClientTest.php

```diff
@@ -93,6 +98,11 @@ public function testSetupParsers()
 
         $parser = $client->getParser('testConfig');
         $this->assertTrue($parser instanceof UrlEncodedParser);
+
+        $unrecognizedParser = 'unrecognizedParser';
```

Comment:
It is better to move it to separate test-method such as `testSetupUnrecognizedParser`.